### PR TITLE
FEATURE: Make login logo configurable

### PR DESCRIPTION
**Pull request by markusguenther**
Since ages, we are able to configure the background of the login to customize. The logo was often replaced by CSS. With this change, we are able to configure a logo as well.

The new setting is  'Neos.Neos.userInterface.backendLoginForm.logoImage'

**Review instructions**

Configure a new logo and check the login screen.

Screenshot 2022-10-18 at 10 39 31

### CODE REVIEW

**Review by jonnitto — APPROVED** [review]

**Review by crydotsnake — APPROVED** [review]
Looks good! 👍🏽 We should also document this new feature.

**Comment by markusguenther** [comment]
> Looks good! 👍🏽 We should also document this new feature.

Good point :)

**Comment by markusguenther** [comment]
Sadly, the whole config is not part of the RST documentation. So, it is a missing piece ;)

**Comment by markusguenther** [comment]
The section is not so huge. So if someone has some free time and wants to add some more docu ;)
https://neos.readthedocs.io/en/8.1/References/Configuration/Configuration.html

**Comment by markusguenther** [comment]
Hope that the formatting is right. Furthermore, please check the documentation @jonnitto and @crydotsnake

**Review by crydotsnake — APPROVED** [review]

**Comment by crydotsnake** [comment]
Looks good!

This gives me an idea. Maybe, we could integrate a check for RST formatting into our GitHub Actions infrastructure.

**Comment by markusguenther** [comment]
Ok a linter would give us maybe some benefits, but we need to resolve a lot of issues first ;)
And make `rst-lint` work with `toctree` from `spinx`.
Screenshot 2022-10-18 at 19 06 40

**Review by Sebobo — APPROVED** [review]

File: Neos.Neos/Resources/Private/Fusion/Backend/Views/Login.fusion [hunk 1]
```diff
@@ -173,3 +161,21 @@ prototype(Neos.Neos:Component.Login.Form) < prototype(Neos.Fusion:Component) {
     </Neos.Fusion.Form:Form>
   `
 }
+
+prototype(Neos.Neos:Component.Login.ImageSource) < prototype(Neos.Fusion:Component) {
```
Comment (Sebobo) [inline Neos.Neos/Resources/Private/Fusion/Backend/Views/Login.fusion]: Nitpick: move to a separate file ;)

Comment (jonnitto) [reply Neos.Neos/Resources/Private/Fusion/Backend/Views/Login.fusion]: Good point. @markusguenther Can you do this?

Comment (mhsdesign) [reply Neos.Neos/Resources/Private/Fusion/Backend/Views/Login.fusion]: i would name it `Neos.Neos:View.Login.ImageSource` and keep it there 🙈 or just merge it as is

there are parts of `Neos.Neos` that are much dirtier - which need our attention ^^

Comment (crydotsnake) [reply Neos.Neos/Resources/Private/Fusion/Backend/Views/Login.fusion]: I would also be in favor of outsourcing this part to an external component.

Comment (markusguenther) [reply Neos.Neos/Resources/Private/Fusion/Backend/Views/Login.fusion]: Done

**Pull request merged**